index.html

```diff
@@ -3,19 +3,34 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
+    <title>TV Show Project | Jacint M. David (Jacint-M-David)</title>
     <link href="style.css" rel="stylesheet" />
     <link rel="icon" href="/favicon.ico" type="image/x-icon" />
   </head>
 
   <body>
+    <fieldset>
+      <div class="inp">
+        <label for="selectEpisode">Select Episode</label>
+        <select onChange="selectEpisode()" name="selectEpisode" id="selectEpisode">
```

Comment:
Also, in addition to what @berkeli said, you will want to use event listeners rather than inline event declarations for better separation of your presentation vs functionality.